src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        FigureSupplier figureSupplier = new FigureSupplier();
+        Figure [] figures = new Figure[6];
+        for (int i = 0; i < figures.length; i++) {
+            if (i < 5) {
```

Comment:
![image](https://github.com/mate-academy/jv-oop-advanced/assets/20326858/6f2f14ab-d0d1-4430-a45f-3f1c49a82df0)
```suggestion
            if (i < figures.length / 2) {
```